kubernetes-elasticsearch-cluster/README.md

```diff
@@ -1,3 +1,4 @@
-The configs here were taken from the [kubernetes-elasticsearch-cluster](https://github.com/pires/kubernetes-elasticsearch-cluster)
-repository with some modifications for our use case such as internal load
-balancing.
\ No newline at end of file
+The configs here were taken from the kubernetes-elasticsearch-cluster repository with some modifications:
+- Using [StatefulSet](https://github.com/pires/kubernetes-elasticsearch-cluster/tree/master/stateful) for persistent storage
+- Be able to support large datasets/indices
```

Comment:
Actually, maybe delete this? I guess it's deploy.sh that has the large dataset stuff.

Reply:
Ya makes sense done.